_topic_map.yml

```diff
@@ -1041,11 +1041,13 @@ Topics:
     File: planning-migration-3-to-4
   - Name: Migrating OpenShift Container Platform 3.7 to 4.2
     File: migrating-openshift-3-to-4
-- Name: Migrating OpenShift Container Platform 4.1 to 4.2
+- Name: Migrating OpenShift Container Platform 4 to 4
```

Comment:
I'm not as concerned about this 4 -> 4, but could always consider doing 4.x -> 4.x

Reply:
agree